deploy/amp/database_initialise.php

```diff
@@ -5,12 +5,12 @@
 <body>
     <?php
 
-        recreate_database('buttonmen');
-        recreate_database('buttonmen_test');
-
-        $conn = db_connect('buttonmen');
-        run_all_sql_files($conn);
+// james: uncomment the following lines to also initialise the interactive database
```

Comment:
Nit: it's obvious to me what you mean by "interactive database", but i'm not sure it would be obvious to someone just starting out development, who's the target audience for this comment.  I'd say something more like "to (re)initialize the primary 'buttonmen' database", or something.  Don't feel strongly about the wording --- i mostly think "interactive" is confusing because "interactive database" could also mean "interactive PHP interface to the database".